### ISSUE TITLE
Error: ENOENT, no such file or directory

### ISSUE THREAD

**Issue by npabisz** [igorklopov/enclose#64]
So I'm using module which loads some templates from directory, how should I define config for compilator?

```
var Steam = require('steam');
```

If you compile and run this it throws error. 

```
Error: ENOENT, no such file or directory
    at Error (native)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+s+e+r+v+e+r+/+n+o+d+e+/+s+t+e+a+m+/+n+o+d+e+_+m+o+d+u+l+e+s+/+s+t+e+a+m+/+n+o+d+e+_+m+o+d+u+l+e+s+/+s+t+e+a+m+-+r+e+s+o+u+r+c+e+s+/+s+t+e+a+m+_+l+a+n+g+u+a+g+e+_+p+a+r+s+e+r+/+i+n+d+e+x+.+j+s.dev (<anonymous>)
    at r (<anonymous>)
    at N (<anonymous>)
    at f (<anonymous>)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+s+e+r+v+e+r+/+n+o+d+e+/+s+t+e+a+m+/+n+o+d+e+_+m+o+d+u+l+e+s+/+s+t+e+a+m+/+n+o+d+e+_+m+o+d+u+l+e+s+/+s+t+e+a+m+-+r+e+s+o+u+r+c+e+s+/+i+n+d+e+x+.+j+s.dev (<anonymous>)
    at r (<anonymous>)
    at N (<anonymous>)
    at f (<anonymous>)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+s+e+r+v+e+r+/+n+o+d+e+/+s+t+e+a+m+/+n+o+d+e+_+m+o+d+u+l+e+s+/+s+t+e+a+m+/+l+i+b+/+s+t+e+a+m+_+c+l+i+e+n+t+.+j+s.dev (<anonymous>)
```

These templates are located at ./node_modules/steam/node_modules/steam-resources/steam_language/*. However if I define config like this: 

```
module.exports = {
    assets: "./node_modules/steam/node_modules/steam-resources/steam_language/*"
};
```

It still throws error, how should I define this or work around? ;)

**Comment by igorklopov** [comment]
I investigated - this is a bug in EncloseJS itself. I will fix in next release. Thanks!

**Comment by npabisz** [comment]
That's great, can't wait!

**Comment by npabisz** [comment]
Any updates on that? :)

**Comment by igorklopov** [comment]
Compatibility with `steam` fixed in 1.0.0. Please try again.
